### ISSUE TITLE
/bin/sh: 1: cargo: not found

### ISSUE THREAD

**Issue by ppdogg** [chaos-mesh/toda#32]
I was building the project in ssh connected to a remote virtual machine.
when I execute `make`, the error comes with:
```
/bin/sh: 1: cargo: not found
```
I look it up and found this solution works for me: https://stackoverflow.com/questions/49676490/when-installing-rust-toolchain-in-docker-bash-source-command-doesnt-work

I made these changes in `toda/Dockerfile`:
- change `sh` to `bash`
```
RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- --default-toolchain nightly-2021-12-23 -y
```
- modify `ENV` grammer
```
ENV PATH="/root/.cargo/bin:${PATH}"
```

However, in my own computer, which is mac os, I build this project successfully without changing anything. I believe remote virtual machine might encounter this problem.

**Comment by YangKeao** [comment]
I don't think these modification is needed. 

For the first, https://rustup.rs/ told me it supports the `sh`.

For the second, see the "Alternative syntax" in https://docs.docker.com/engine/reference/builder/#env (though, I agree that including the "=" is better :man_facepalming:  )

As the `Dockerfile` are executed inside a container, I don't think there are any difference between a mac os and a virtual machine (actually, the docker will run a virtual machine for mac os). 

I guess there were some network issues, which cause the `rustup` script somehow failed to install the rustc, cargo, etc but doesn't return the error. I'm not sure.

**Comment by ppdogg** [comment]
I agree that building a docker image has nothing to do with host os, but that's the part that confused me. 
I believe the error comes from either a modified centos or a rust installation problem. thanks for replying
